extensions/email/src/main/java/org/mule/extension/email/api/EmailAttributesBuilder.java

```diff
@@ -15,7 +15,6 @@
 import static javax.mail.Message.RecipientType.CC;
 import static javax.mail.Message.RecipientType.TO;
 import static org.mule.extension.email.api.EmailContentProcessor.process;
-
 import org.mule.extension.email.api.exception.EmailException;
 
 import java.time.LocalDateTime;
```

Comment:
yep, a lot of optional parameters for Email, and the builder can abstract the concept of "empty content" much better